### PR TITLE
Create widget_sample3.php

### PR DESCRIPTION
**Pull request by ecappa**
Allowing test different widget with orbit theme.

### CODE REVIEW

File: core/src/plugins/gui.ajax/widget_sample3.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            //"BOOTER_URL": "index.php?get_action=get_boot_conf&goto=default/Organizations",
+            "BOOTER_URL": "../../index.php?get_action=get_boot_conf",
+            "FORCE_REGISTRY_RELOAD": true,
+            "EXT_REP": "\/",
+            "MAIN_ELEMENT": "<?php echo $ajaxPart; ?>",
+            "SERVER_PREFIX_URI": "../../"
+        };
+        document.observe("ajaxplorer:before_gui_load", function (e) {
+            ajaxplorer.currentThemeUsesIconFonts = true;
+            document.documentElement.className += " ajxp_theme_vision";
```
Comment (cdujeu) [inline core/src/plugins/gui.ajax/widget_sample3.php]: Eric, I guess we should read ajxp_theme_orbit there, no ?

Comment (ecappa) [inline core/src/plugins/gui.ajax/widget_sample3.php]: Yes youre right... I played to much with the template engine ;)

Comment (cdujeu) [inline core/src/plugins/gui.ajax/widget_sample3.php]: can you update the PR ?

**Comment by cdujeu** [comment]
I replaced it directly, your PR seems a bit messy :-) But you get the credit of course!

**Pull request closed without merging**